src/components/itemPage/Item.jsx

```diff
@@ -89,8 +90,41 @@ class Item extends Component {
     );
   }
 
+  onlistening = async () => {
+    (async () => {
+      let SERVICE = (await conf).services[0];
+      const response = await fetch(SERVICE);
+      var myjson = await response.json();
+      this.update_seq = myjson.update_seq;
+
+      let params = this.props.match.params;
+      this.eventSource = new EventSource(`${SERVICE}:5984/argos/_changes?filter=argos/item&feed=eventsource&since=${this.update_seq}&corpus=${params.corpus}`);
```

Comment:
> `${SERVICE}:5984/argos`

`SERVICE` is supposed to be the Argos endpoint. 
On your computer it was `http://if05.local:5984/argos`, on the test server it is `http://argos2.test.hypertopic.org`. Relatively to this endpoint, the path is `/_changes ...`.

Reply:
Ah. So the code is not identical for development and production...

Reply:
Isn't that something that can be redefined in the rewrite of Argos?

Reply:
Wouldn't it be easier to create a new constant?

Reply:
> Isn't it something that can be redefined in the rewrite of Argos?

I don't understand: it is already.

> Ah. So the code is not identical for development and production..

It should be. It's your development constant that was erroneous.
You should have used AAAforREST (the reverse proxy with rewrite) in front of Argos.

Reply:
If Argos is installed as stated [here](https://github.com/Hypertopic/Argos/blob/v4/README.md).

`http://localhost/` is equivalent to `http://localhost:5984/argos/_design/argos/_rewrite/`.

Therefore `http://localhost/_changes` is equivalent to `http://localhost:5984/argos/_design/argos/_rewrite/_changes` which is itself equivalent to `http://localhost:5984/argos/_changes`.

Let `SERVICE` be `http://localhost` on development environment and `http://argos2.hypertopic.org`on production, then changes API is available on `${SERVICE}/_changes`.